platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheProblemsServiceIntegTest.groovy

```diff
@@ -69,7 +69,14 @@ class ConfigurationCacheProblemsServiceIntegTest extends AbstractConfigurationCa
             fqid == REGISTRATION_UNSUPPORTED
             contextualLabel == "registration of listener on 'Gradle.buildFinished' is unsupported"
             definition.severity == Severity.WARNING
-            definition.documentationLink != null
+            definition.documentationLink.url.endsWith("/userguide/configuration_cache.html#config_cache:requirements:build_listeners")
+            locations.size() == 2
```

Comment:
:x:  We should stop doing this, as implicit locations (e.g., task location, which I'm trying to introduce) break this.